sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryCompilationErrors.scala

```diff
@@ -1337,10 +1337,11 @@ object QueryCompilationErrors {
       "Stream-stream join without equality predicate is not supported", plan = Some(plan))
   }
 
-  def cannotUseMixtureOfAggFunctionAndGroupAggPandasUDFError(): Throwable = {
+  def cannotUseMixtureOfAggFunctionAndGroupAggPandasUDFError(
+      groupAggPandasUDFNames: Set[String]): Throwable = {
```

Comment:
Can we pass `Seq[String]` instead?

Reply:
You can make it deduplicated by calling `distinct` method without converting it to `Set`.